content/source/docs/enterprise/api/index.html.md

```diff
@@ -33,11 +33,18 @@ Future APIs will increment this version, leaving the `/v1` API intact, though in
 
 ## Paths
 
-All V2 API endpoints use `/api/v2` as the subpath unless otherwise specified.
+All V2 API endpoints use `/api/v2` as a prefix unless otherwise specified.
 
 For example, if the API endpoint documentation defines the path `/runs` then the full path is `/api/v2/runs`.
 
 ## JSON API Formatting
 
-The TFE endpoints use the [JSON API specification](http://jsonapi.org/) which specifies standards for [HTTP error codes](http://jsonapi.org/examples/#error-objects-error-codes), [error objects](http://jsonapi.org/examples/#error-objects-basics), [document structure](http://jsonapi.org/format/#document-structure), [HTTP Request/Response headers](http://jsonapi.org/format/#content-negotiation), and other key aspects of the API.
+The TFE endpoints use the [JSON API specification](http://jsonapi.org/), which specifies standards for [HTTP error codes](http://jsonapi.org/examples/#error-objects-error-codes), [error objects](http://jsonapi.org/examples/#error-objects-basics), [document structure](http://jsonapi.org/format/#document-structure), [HTTP Request/Response headers](http://jsonapi.org/format/#content-negotiation), and other key aspects of the API.
 
+### Query Parameters
+
+Although most of our API endpoints use the POST method and receive their parameters as a JSON object in the request payload, some of them use the GET method. These GET endpoints sometimes require URL query parameters, in the standard `...path?key1=value1&key2=value2` format.
+
+Since these parameters were originally designed as part of a JSON object, they sometimes have characters that must be [percent-encoded](https://en.wikipedia.org/wiki/Percent-encoding) in a query parameter. For example, `[` becomes `%5B` and `]` becomes `%5D`.
```

Comment:
As note, if you quote the URL `"https://app.terraform.io/"` and escape (`\`) the `[]` characters that will also work. As long as the examples always specify a single way of doing it it'll be fine. Wondering whether escaped characters would provide to be more legible in the examples rather than the encoded ones.

Reply:
Isn't that a curl-specific way of asking to have encoding taken care of for you? In what's supposed to be a general-purpose reference, I think that's likely to mislead.

Reply:
It could be `curl` related, but it is how our documentation is referenced. In a library you usually would not think of encoding yourself as it would handle it for you. You could pass this as a map of things you want the library to pass on.